hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/BackendEntityEnricher.java

```diff
@@ -72,6 +78,14 @@ public void enrichTrace(StructuredTrace trace) {
         .forEach(pair -> decorateWithBackendEntity(pair.getRight().get(), pair.getLeft(), trace));
   }
 
+  @Override
+  public void enrichEvent(StructuredTrace trace, Event event) {
+    String backendOperation = SpanAttributeUtils.getFirstAvailableStringAttribute(event, BACKEND_OPERATIONS);
```

Comment:
We are missing handling `http` or `grpc` backend.
For, http_backend, we have to enrich `get/post` method.

Reply:
we can add `http.method` and `grpc.method` in a list.

Reply:
could it possible that http.method present as a tag, but identified backend was sql? if so, considering this is an order list, it may possible that it will have an operation as `GET` instead of `select`